independent-publisher-connectors/Shopify Admin Content/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2895 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Shopify Admin Content",
+    "description": "The Shopify REST Admin API lets you build apps and other integrations for the Shopify admin.",
```

Comment:
@thatapiguy This description seems vague and can please provide some more information to understand the connector better 

Documentation - [Description of connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#description)